### ISSUE TITLE
UI test fails in personalSettings suite on Edge browser

### ISSUE THREAD

**Issue by phil-davis** [owncloud/core#30630]
### Steps to reproduce
1. Run UI test suite ``personalSettings`` on Microsoft Edge browser
2. See 2 failed scenarios

### Expected behaviour
Test suite passes

### Actual behaviour
Tests fail regularly in the Travis cron nightly job:
```
  Scenario: Change password                                                 # /home/[secure]/build/owncloud/core/tests/ui/features/personalSettings/changePasswordFromSetting.feature:15
    When I change the password to "passcode"                                # PersonalGeneralSettingsContext::iChangeThePasswordTo()
      Form field with id|name|label|value|placeholder "pass2" not found.
```

```
--- Failed scenarios:
    /home/[secure]/build/owncloud/core/tests/ui/features/personalSettings/changePasswordFromSetting.feature:15
    /home/[secure]/build/owncloud/core/tests/ui/features/personalSettings/changePasswordFromSetting.feature:20

```

### Server configuration
**Operating system**:
Configuration as per current Travis cron job against ``stable10``

**Comment by phil-davis** [comment]
Sample test output:
https://travis-ci.org/owncloud/core/jobs/346373309

**Comment by phil-davis** [comment]
Demonstration of now-passing tests in #30361 
Fix in #30362

**Comment by phil-davis** [comment]
ToDo:
- [ ] look in the next nightly Travis cron run and see if it passes
